### DISCUSSION TITLE
[Question] 关于 event 的使用方式的一些问题

### DISCUSSION THREAD

**Discussion by zhuzeyu22** [opensumi/core#3292] [General]
event 的使用方式

有些地方使用的是 this.eventBus.fire 出来的事件
有些地方是 Emitter 出来的事件发射器，

我想知道这两种不同的使用方式分别什么情况下使用比较好？

**Comment by erha19** [comment]
如果需要解耦服务间调用，可以用 this.eventbus, 否则都可以

**Comment by zhuzeyu22** [comment]
> 如果需要解耦服务间调用，可以用 this.eventbus, 否则都可以

也就是用 this.eventbus 是一种比较好的方式，好的懂了。
